client/utils/auth.py

```diff
@@ -7,6 +6,9 @@
 import sys
 import time
 import webbrowser
+from urllib.request import urlopen
+import json
+from .html import auth_html, partial_course_html, partial_nocourse_html, red_css
```

Comment:
That makes sense. I had just never seen an import start with a dot before.
No change necessary.

On Sun, Jun 21, 2015 at 6:45 PM, Alvin Wan <notifications@github.com> wrote:

> In client/utils/auth.py
> <https://github.com/Cal-CS-61A-Staff/ok-client/pull/76#discussion_r32901297>
> :
>
> > @@ -7,6 +6,9 @@
> >  import sys
> >  import time
> >  import webbrowser
> > +from urllib.request import urlopen
> > +import json
> > +from .html import auth_html, partial_course_html, partial_nocourse_html, red_css
>
> I originally tried putting all HTML templates in a directory called
> "templates," but it wasn't included in the binary, so I created a new
> python file html.py with all the html contents. The prepending dot is the
> current directory. Should I add a comment, or is should I rename it
> "templates.py".?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Cal-CS-61A-Staff/ok-client/pull/76/files#r32901297>.
>